packages/launchpad/package.json

```diff
@@ -45,6 +45,7 @@
   "dependencies": {
     "@cosmjs/crypto": "^0.22.2",
     "@cosmjs/encoding": "^0.22.2",
+    "@cosmjs/launchpad-ledger": "^0.22.2",
```

Comment:
I'd prefer to make Ledger support optional, as it comes with dependencies that can be annoying sometimes (compiling libusb C code for Node.js). Can we swap the dependencies and more the demo to @cosmjs/launchpad-ledger?